etc/cin.profile

```diff
@@ -18,15 +18,15 @@ ipc-namespace
 net none
 nodbus
 nodvd
-nogroups
-nonewprivs
+#nogroups
+#nonewprivs
 notv
 noroot
 protocol unix
-seccomp
+#seccomp
```

Comment:
I'm not sure we should disable some of these (especially `seccomp`). @SkewedZeppelin any thoughts?

Reply:
Agreed.

Reply:
rendertime 

13:55  with     seccom
13:05  without it